app/models/recipe_food.rb

```diff
@@ -0,0 +1,8 @@
+class RecipeFood < ApplicationRecord
+  belongs_to :recipe
+  belongs_to :food
+
+  def calc_value(quantity, price)
+    quantity * price
+  end
+end
```

Comment:
- [ ] Please validate the input fields for your forms.

Reply:
Validations have been added to required places!